src/app/shell/personal-cabinet/applications/application-card/application-card.component.ts

```diff
@@ -55,11 +65,22 @@ export class ApplicationCardComponent implements OnInit {
   }
 
   /**
-   * This method emit on deny action
+   * This method emit reject Application
    * @param Application application
    */
-  onReject(application: Application): void {
-    this.rejected.emit(application);
+    onReject(application: Application): void {
+    const dialogRef = this.matDialog.open(RejectModalWindowComponent, {
+      width: '330px',
+      data: {
```

Comment:
I belive you just can remove it if you're not passing any data there?

Reply:
removed it